src/components/consul/selectors/ImageSelector.js

```diff
@@ -103,6 +108,26 @@ export const ImageSelector = ({
       }
     }
 
+    if (selectorType === IMAGE_SELECTOR_TYPES.SUE) {
+      const { GPSLatitude, GPSLongitude } = imagesAttributes[index].exif || {};
+      const { selectedPosition, setValue = () => {} } = coordinateCheck || {};
+
+      if (
+        !!GPSLatitude &&
+        !!GPSLongitude &&
+        selectedPosition.latitude === GPSLatitude &&
+        selectedPosition.longitude === GPSLongitude
```

Comment:
why are there the two checks of presence before the === checks?

Reply:
Actually unnecessary. There is no need to double check here. The === check will be enough. 👍

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/b94d017584b4fcd1ad1be4b4f781619dbe741f1a